src/v2/Apps/_Preferences2/PreferencesApp.tsx

```diff
@@ -71,6 +71,8 @@ export const PreferencesApp: FC<PreferencesAppProps> = ({ viewer }) => {
               variables: { input: { subscriptionGroups } },
             })
 
+            formikHelpers.setTouched({})
```

Comment:
Why not use https://formik.org/docs/api/formik#resetform-nextstate-partialformikstatevalues--void ?